tutorial/airfoilopt/mesh/genMesh.py

```diff
@@ -49,7 +49,7 @@
     # ---------------------------
     "ps0": -1.0,
     "pGridRatio": -1.0,
-    "cMax": 3.0,
+    "cMax": 1.0,
```

Comment:
I would prefer if default values are removed from this, and instead we just link to the options page.

Reply:
We have already linked to the options page in the doc, I think it is sufficient to just remove this line instead of setting it to 1.0

Reply:
I was referring to all the options actually, since I think it may be confusing for beginners to be given so many options when in fact they are all just defaults.

Reply:
Ahh, I see, I agree with this, it seems all of the important options are not defaults. I will get rid of the default parameters that are definitely confusing.